src/components/ChatBubble/index.js

```diff
@@ -1,16 +1,32 @@
 import React from 'react';
+import { View, Text } from 'react-native';
 import { Bubble } from 'react-native-gifted-chat';
 import styles from './styles';
 
-const ChatBubble = props => (
-  <Bubble
-    {...props}
-    textStyle={{ left: styles.text, right: styles.text }}
-    wrapperStyle={{
-      left: styles.backgroundGray,
-      right: styles.backgroundYellow,
-    }}
-  />
-);
+const ChatBubble = props => {
+  const {
+    currentMessage: { createdAt },
+    position,
+  } = props;
+
+  const date = new Date(createdAt).toLocaleString();
+  const [, time, suffix] = date.split(' ');
+  const [hours, minutes] = time.split(':');
+  const timeStamp = `${hours}:${minutes} ${suffix}`;
+
+  return (
+    <View>
+      <Bubble
+        {...props}
+        textStyle={{ left: styles.text, right: styles.text }}
+        wrapperStyle={{
+          left: [styles.backgroundGray, styles.paddingVertical],
+          right: [styles.backgroundYellow, styles.paddingVertical],
```

Comment:
You can add this two styles in just one for example `styles.rightBubble` and add the background coloa and the padding